test/functional/p2p_filter.py

```diff
@@ -125,8 +143,11 @@ def run_test(self):
 
         self.log.info('Check that we receive a tx in reply to a mempool msg if the filter matches a mempool tx')
```

Comment:
same commit:

This comment is obviously wrong (`git blame myself`)

```suggestion
        self.log.info('Check that we receive a tx if the filter matches a mempool tx')
```

Reply:
unrelated nit in commit facd5f9765:

Has this comment been fixed?